docs/src/app/(public)/(content)/react/handbook/forms/page.mdx

```diff
@@ -0,0 +1,523 @@
+# Forms
+
+<Subtitle>A guide to building forms with Base UI components.</Subtitle>
+<Meta name="description" content="A guide to buildling forms with Base UI components." />
```

Comment:
Nitpick: Is there a benefit in providing a meta field with the same content as the page subheading?
P.S.: I'd reckon that `Subtitle` should render an `h2` or we should use a slightly different component. 🤔